### PR TITLE
updating perl-vcftools-vcf

### PR DESCRIPTION
**Pull request by Cashalow**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

I have put the version to 0.1.14 because that is what is stated on the Vcf.pm file (0.1.14-12 exactly, the version of the package used to be extracted from there) although the version of the full package is 0.1.15

### CODE REVIEW

**Comment by Cashalow** [comment]
To fix the cpanm error on osX, I have used the fix presented here : https://github.com/bioconda/bioconda-recipes/pull/4798, please tell me if it's not okay !

File: recipes/perl-vcftools-vcf/build.sh [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
-
 mkdir -p perl-lib/lib
 cp src/perl/*.pm perl-lib/lib
-# Fix version to something cpan installable tools like
-sed -i.bak 's/r953/0.953/' perl-lib/lib/Vcf.pm
 # Fix naming scheme of package to make Builder happy
 sed -i.bak 's/Vcf.pm.  Module/Vcf - Module/' perl-lib/lib/Vcf.pm
 cp $RECIPE_DIR/Build.PL perl-lib
 cd perl-lib
-cpanm -v -i .
```
Comment (bgruening) [inline recipes/perl-vcftools-vcf/build.sh]: Why can you not call Build.PL directly? https://github.com/bioconda/bioconda-recipes/pull/4798/files#diff-07b0d6ce7a6e79394d4384e7db728326R4

My fear is that cpanm does more magic things, like pulling down other libs that should be defined more explicitly.

Comment (Cashalow) [reply recipes/perl-vcftools-vcf/build.sh]: Indeed it works perfectly well thanks ! I have removed the cpan dependency accordingly.

**Comment by peterjc** [comment]
Looks good to me, and you've address the issues flagged by @bgruening - merging...

**Comment by peterjc** [comment]
Thanks!

**Pull request merged**